src/hotspot/os/linux/os_linux.cpp

```diff
@@ -1827,6 +1848,18 @@ void * os::Linux::dlopen_helper(const char *filename, char *ebuf,
     event.set_errorMessage(nullptr);
     event.commit();
 #endif
+
+#ifndef IA32
+    // Quickly test to make sure denormals are correctly handled.
```

Comment:
Nit: I recommend using "subnormal" rather than "denormal" for general terminology on this point. While "denormal" was used in the original IEEE 754 standard from 1985, subsequent iterations of the standard using "subnormal" The term "subnormal" has also been used for the last several editions of JLS and JVMS.

Reply:
I've long avoided "subnormal" because

subnormal in British English, adjective
3.  [old-fashioned, offensive] a person of low intelligence

Reply:
All done. Good to go?